serializer/seata-serializer-seata/src/main/java/io/seata/serializer/seata/SeataSerializer.java

```diff
@@ -16,52 +16,133 @@
  */
 package io.seata.serializer.seata;
 
-import java.nio.ByteBuffer;
-
 import io.netty.buffer.ByteBuf;
 import io.netty.buffer.Unpooled;
 import io.seata.common.loader.LoadLevel;
+import io.seata.common.loader.Scope;
 import io.seata.common.util.BufferUtils;
 import io.seata.core.protocol.AbstractMessage;
+import io.seata.core.protocol.ProtocolConstants;
 import io.seata.core.serializer.Serializer;
 
+import java.nio.ByteBuffer;
+
 /**
  * The Seata codec.
- *
  */
-@LoadLevel(name = "SEATA")
+@LoadLevel(name = "SEATA", scope = Scope.PROTOTYPE)
```

Comment:
为什么它是一个多例？
Why is it a PROTOTYPE case?

Reply:
因为它的构造方法是带有参数的所以不能是单例，所以上次提过new出来之后会有map管理
Because its constructor method is with parameters so can not be singleton, so as mentioned last time, it will be managed by the map after new obj.